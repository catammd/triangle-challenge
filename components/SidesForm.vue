<template>
    <div>Sides form</div>
</template>

<script>
export default {

}
</script>

<style>

</style>
