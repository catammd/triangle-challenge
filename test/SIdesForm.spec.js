import { shallowMount } from '@vue/test-utils'
import SidesForm from '@/components/SidesForm'

describe('SidesForm', () => {
  it('renders the component', () => {
    // arrange
    const wrapper = shallowMount(SidesForm)

    // assert
    expect(wrapper.html()).toMatchSnapshot()
  })
})
