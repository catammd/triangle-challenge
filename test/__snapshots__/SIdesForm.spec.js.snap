// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidesForm renders the component 1`] = `"<div>Sides form</div>"`;
